Index: Schedule.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n#include \"Schedule.h\"\r\n\r\nSchedule::Schedule(vector<UniClass> classes) {\r\n    this -> classes = classes;\r\n}\r\n\r\nbool Schedule::addClass(UniClass classAdd){\r\n    return false;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Schedule.cpp b/Schedule.cpp
--- a/Schedule.cpp	
+++ b/Schedule.cpp	
@@ -1,4 +1,3 @@
-
 #include "Schedule.h"
 
 Schedule::Schedule(vector<UniClass> classes) {
